client/blocks/reader-full-post/carousel.scss

```
@@ -0,0 +1,154 @@
+/* Post carousel block styles */
+
+.wp-block-newspack-blocks-carousel {
+	position: relative;
+	margin-top: 0;
+
+	div {
+		max-width: 100%;
+		padding: 0;
+		position: relative;
+		margin-bottom: 0;
+		word-break: break-word;
+		overflow-wrap: break-word;
+		a {
+			color: var(--studio-white);
+
+			&:active,
+			&:focus,
+			&:hover {
+				color: var(--studio-white);
+			}
+		}
+		.entry-title {
+			font-size: $font-title-small;
+
+			a {
+				-webkit-box-orient: vertical;
+				display: -webkit-box;
+				-webkit-line-clamp: 3;
+				max-height: 3.5625em;
+				overflow: hidden;
+				text-overflow: ellipsis;
+			}
+		}
+		.avatar {
+			height: 1.8em;
+			width: 1.8em;
+		}
+
+		.entry-wrapper {
+			bottom: 0;
+			background-color: rgba(var(--color-text-rgb), 0.5);
+			color: var(--studio-white);
+			left: 0;
+			padding: 1.5em;
+			position: absolute;
+			right: 0;
+		}
+		.entry-meta {
+			color: var(--studio-white);
+			margin-bottom: 0;
+			font-size: $font-body-small;
+
+			a {
+				color: var(--studio-white);
+				font-weight: bold;
+				text-decoration: none;
+
+				&:active,
+				&:focus,
+				&:hover {
+					color: rgba(var(--studio-white), 0.75);
+				}
+			}
+		}
+	}
+	.post-thumbnail {
+		margin: 0;
+		height: 100%;
+		width: 100%;
+		padding: 0;
+		position: relative;
+		a,
+		img {
+			display: block;
+			height: 100%;
+			object-fit: cover;
+			width: 100%;
+
+			&.image-fit-contain {
+				object-fit: contain;
+			}
+		}
+	}
+	&__placeholder {
```

Nit: can we please use the full extension here so simple _Find_ operations on the selector will work?
```suggestion
	.wp-block-newspack-blocks-carousel__placeholder {
```